inscriptions/ISic000387.xml

```diff
@@ -185,14 +190,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/>SAC
+                    <lb n="2"/>O R
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text of CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#JH">
```

Comment:
again, no point / need. Probably this is sacrum or similar, but this would be reserved for the commentary. But it coudl be sacerdos, etc., etc.